build/commands/lib/build.js

```diff
@@ -132,7 +132,11 @@ const build = (buildConfig = config.defaultBuildConfig, options) => {
   if (config.xcode_gen_target) {
     util.generateXcodeWorkspace()
   } else {
-    util.buildTarget()
+    util.buildTarget().catch(err => {
```

Comment:
requiring this everywhere `buildTarget` is called is definitely not ideal and I think there is already another buildTarget that this is missing now. In particular `options.continueOnFail` should be encapsulated within `buildTarget`